src/TableView.elm

```diff
@@ -193,27 +185,22 @@ appendFirstRowCell dateWithSessions timeDelimiters model numColumns index column
         else
             case sessionStarting of
                 Just sessionStarting ->
-                    td [ rowspan rowSpanVal, colspan colSpanVal ]
+                    td [ class "prog-cell", rowspan rowSpanVal, colspan colSpanVal ]
                         [ div []
-                            [ a [ href ("/events/" ++ model.eventId ++ "/sessions/" ++ (toString sessionStarting.id)) ]
-                                [ text
-                                    (sessionStarting.name
-                                        ++ "  "
-                                        ++ "Chair:  "
-                                        ++ sessionStarting.chair
-                                        ++ "  "
-                                        ++ "Location:  "
-                                        ++ sessionStarting.location
-                                        ++ "  "
-                                        ++ (DateUtils.displayTimeOfDay sessionStarting.startTime)
-                                        ++ " - "
-                                        ++ (DateUtils.displayTimeOfDay sessionStarting.endTime)
-                                    )
+                            [ span [ class "prog-cell__header" ]
+                                [ a [ class "prog-cell__name", href ("/events/" ++ model.eventId ++ "/sessions/" ++ (toString sessionStarting.id)) ]
+                                    [ text (sessionStarting.name)
+                                    ]
+                                , button [ hidden publishOrPreviewUi, class "prog-cell__action", onClick (DeleteSession sessionStarting.id), style [ ( "margin-left", "0.2rem" ) ] ] [ text "delete" ]
```

Comment:
wo wo wo wo! don't inline styles on the elements. You're making more work for yourself by separating the styles of an element in to more than one place. Please move this into the class css declaration